docs/TargetMaintenance/CreatingAUnifiedTarget.md

```diff
@@ -64,6 +66,6 @@ These instructions explain how to create a Unified Target configuration for an e
 
 - re-start CLI (Disconnect / Connect), then do a `diff all bare`, save the output into a file named `<manufacturer_id>-<board_name>.config` with 'Save to File'. It is crucial that the name exactly matches the manufacturer id and board name specified in the file, or else checking of the pull request you are going to open in a subsequent step will fail;
 
-- edit the resulting file and verify that there are no extra lines before the one starting with `# Betaflight`, and no extra lines after the last line starting with `set`;
+- edit the resulting file and verify that the first line in the file starts with `# Betaflight`, otherwise delete the lines before this line. Likewise, there must be no extra lines after the last line starting with `set`, delete all subsequnet lines otherwise;
```

Comment:
I don't think this makes it clear now that the banner line (beginning with `# Betaflight`) should be left untouched.

Reply:
ok